gramps/gui/editors/displaytabs/embeddedlist.py

```diff
@@ -533,6 +534,13 @@ def build_columns(self):
             self.columns.append(column)
             self.tree.append_column(column)
         self.track_ref_for_deletion("columns")
+        # associate the selector config name to the PersistentTreeView
+        self.tree.set_config_name(self.get_config_name())
+        self.uistate.connect('font-changed', self.tree.restore_column_size)
```

Comment:
This belongs in the `PersistentTreeView` class.  We will always need it.

Reply:
Yes, I know but the PersistentTreeView doesn't know the self.uistate and the self.tree.
It is a Gtk widget subclass with new methods.